docs/reference/api.md

```diff
@@ -281,7 +281,7 @@ addStuff().then((d) => console.log(d)); // 13 (5 + 7 + 1)
 | `challenge.handshake`                  | **DEPRECATED**                                       | Use `discovery.wallet` instead.|
 | `discovery.authn.endpoint`             | `https://fcl-discovery.onflow.org/api/testnet/authn` | Endpoint for alternative configurable Wallet Discovery mechanism. Read more on [discovery](./api.md#discovery)                                                                                         |
 | `discovery.wallet` **(required)**      | `https://fcl-discovery.onflow.org/testnet/authn`     | Points FCL at the Wallet or Wallet Discovery mechanism.     
-| `discovery.wallet.method`     | `IFRAME/RPC`, `POP/RPC`, `TAB/RPC`, or `HTTP/POST`     | Describes which service strategy a wallet should use.                                                                                                                                      |
+| `discovery.wallet.method`     | `IFRAME/RPC`, `POP/RPC`, `TAB/RPC`, `HTTP/POST`, or `EXT/RPC`     | Describes which service strategy a wallet should use.                                                                                                                                      |
 | `fcl.limit`                         | `100`                                            | Specifies fallback compute limit if not provided in transaction.  Provided as integer.                                                                                      |
 | `flow.network`                         | `testnet`                                            | Used in conjunction with stored interactions and provides FCLCryptoContract address for `testnet` and `mainnet`. Possible values: `local`, `canarynet`, `testnet`, `mainnet`.                                                                            |
```

Comment:
Should we add "Recommended" to this as it's used by so much now. Or maybe make it a required?

Reply:
Well only recommended if pointing directly, but we could make that clear.

Reply:
Sorry, I mean add recommended to `flow.network`

Reply:
Added